src/main/java/org/checkerframework/checker/noliteral/NoLiteralChecker.java

```diff
@@ -0,0 +1,22 @@
+package org.checkerframework.checker.noliteral;
+
+import org.checkerframework.common.basetype.BaseTypeChecker;
+import org.checkerframework.framework.source.SuppressWarningsKeys;
+
+/** The NoLiteral Checker enforces that literals don't go where they don't belong */
+@SuppressWarningsKeys("noliteral")
+public class NoLiteralChecker extends BaseTypeChecker {
+
+  /**
+   * Overridden so that unchecked code defaults are always used for bytecode, because we only want
+   * to enforce @NonConstant if a stub file is present.
```

Comment:
I think the point of this is to prevent defaulting to `@NonConstant`.  If so, please say that.